BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -166,6 +165,20 @@ def integrate(register: true, create_tool_config: false, multi_version: false)
         })[partial]
       end
 
+      # Write out a help file for this Boutiques task
+      helpFileName = name + "_help.html" 
```

Comment:
That's what I'm saying. The BrainPortal app might be running as a user that cannot write to the file system where its code is.

Normally, such 'additions' to the code base is done through rake tasks.